dipy/utils/parallel.py

```diff
@@ -42,17 +46,21 @@ def paramap(
     engine : str
         {"dask", "joblib", "ray", "serial"}
         The last one is useful for debugging -- runs the code without any
-        parallelization. Default: "joblib"
+        parallelization. Default: "ray"
```

Comment:
I'd remove altogether the `Default: ...` part: it is already documented in the method signature, and reduces maintenance burden, no risk to make them go out of sync if the signature changes, etc.

Reply:
Done in 444503737